packages/styles/src/solid-styles.css

```diff
@@ -10,3 +10,14 @@
 @import url('./modules/status-badge.css');
 /* plop:style */
 @import url('./modules/prose.css');
+
+body {
+  --tw-blur: ;
+  --tw-brightness: ;
+  --tw-contrast: ;
+  --tw-grayscale: ;
+  --tw-hue-rotate: ;
+  --tw-invert: ;
+  --tw-saturate: ;
+  --tw-sepia: ;
+}
```

Comment:
That looks odd. 😆 

I'm irritated, that there's no fallback in the classes or when applying. Can you have more a look there?

And if we have to do it like this, I would maybe go with `:root`. But I believe, this shouldn't be done like this....